src/devices/CpuTemperature/README.md

```diff
@@ -1,11 +1,13 @@
 # Cpu Temperature
 
-Device bindings for the CPU Temperature Sensor
+Device bindings for the CPU Temperature Sensor on Linux and Windows
```

Comment:
since we now cover all OS-es (I assume OSX also works through Linux implementation) I'd just not change this